src-ghc/Pact/GasModel/Utils.hs

```diff
@@ -63,7 +63,6 @@ import Bound (abstract, Scope)
 import Control.Exception (throwIO)
 import Data.Aeson (toJSON, ToJSON(..))
 import Data.Default (def)
-import Data.List.NonEmpty (NonEmpty(..))
```

Comment:
Why does such an extensive change to the gas tests need to be made in this PR? wouldn't the gas tests for capability guards be a rather minimal addition, and then we could make this change to drop NonEmpty in a separate PR?

Reply:
It's just a "fix it when you see it" situation. I generally include test improvements in features as opposed to dedicated PRs. Finally there are more improvements than just `NonEmpty`, and the new capability tests leverage that.